src/test/java/dk/sdu/se_f22/productmodule/management/ProductTest.java

```diff
@@ -24,14 +24,14 @@ void setup(){
     
```

Comment:
Is there a reason for this class to exist, since I assume all the code has been moved to `BaseProductTest.java`

Reply:
It exists only because there was an intermediate push from the time this PR was created to now. Because of that push Git sees this file as still existing (the group added functionality to the test) i preferred to let the group sort out was moving to the new class. To make sure i did not cause trouble with their tests.
The group that owns the tests therefore have the responsibility to delete this class

Reply:
we have no write access so i'm gonna have to do something super scuffed

Reply:
[https://github.com/LangeHenrik/SDU-SE-F22/pull/197](url)